### PR TITLE
✨Source Google Search Console: add new streams

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
Adding new streams:
[new stream Keyword Site Report by page](https://github.com/airbytehq/airbyte/issues/26589)
[new stream Keyword Page Report](https://github.com/airbytehq/airbyte/issues/26588)
[new stream Site Report by page](https://github.com/airbytehq/airbyte/issues/26592)
[new stream Site Report by site](https://github.com/airbytehq/airbyte/issues/26594)
[new stream Page Report](https://github.com/airbytehq/airbyte/issues/26591)

## How
For streams based on Keyword, uses class SearchAppearance, that gets records by `SearchAppearance` dimension, value of this dimension uses in filtering after.
Site Report and Page Report use appropriate dimensions and aggregation type.

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`247468320c`](https://github.com/airbytehq/airbyte/pull/27831/commits/247468320c51ce8e90c7e21040ee3d517d4fa336)) - ✅

⏲️  Total pipeline duration: 16mn06s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-google-search-console_add-new-streams/1688038130/247468320c51ce8e90c7e21040ee3d517d4fa336/source-google-search-console/1.1.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Review by lazebnyi — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-search-console/Dockerfile [hunk 1]
```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.1.0
+LABEL io.airbyte.version=1.1.1
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-google-search-console/Dockerfile:15]: Need update minor version.

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`4aac5bfa5b`](https://github.com/airbytehq/airbyte/pull/27831/commits/4aac5bfa5b404fd96e632ed41920e3ee5380e2eb)) - ❌

⏲️  Total pipeline duration: 15mn17s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-google-search-console_add-new-streams/1688113007/4aac5bfa5b404fd96e632ed41920e3ee5380e2eb/source-google-search-console/1.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-search-console test report (commit [`d7d63348a2`](https://github.com/airbytehq/airbyte/pull/27831/commits/d7d63348a231a495a1a8d74882598a13615a35f2)) - ❌

⏲️  Total pipeline duration: 14mn19s 

| Step                                                                                | Result   |
|:------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-search-console/metadata.yaml | ✅        |
| Connector version semver check                                                      | ✅        |
| Connector version increment check                                                   | ✅        |
| QA checks                                                                           | ✅        |
| Code format checks                                                                  | ✅        |
| Connector package install                                                           | ✅        |
| Build source-google-search-console docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                          | ✅        |
| Acceptance tests                                                                    | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/daryna_source-google-search-console_add-new-streams/1688114006/d7d63348a231a495a1a8d74882598a13615a35f2/source-google-search-console/1.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-search-console test
```

**Pull request merged**